### DISCUSSION TITLE
Can't get passing additional flags to gotenberg to work

### DISCUSSION THREAD

**Discussion by lennartb-** [gotenberg/gotenberg#703] [Q&A] [answered]
Hi,
I'm trying to change the UNO listener start timeout value as per https://gotenberg.dev/docs/configuration#libreoffice

However, not matter how I try to pass the parameter, I keep getting an error from tini:
```
docker run --rm -p 3000:3000 gotenberg/gotenberg:7 --uno-listener-start-timeout=20s 
[FATAL tini (6)] exec --uno-listener-start-timeout=20s failed: No such file or directory
```
I'm using Windows Terminal/Powershell if that matters.

I tried escaping with quotes, i.e. `"--uno-listener-start-timeout=20s"`, single quotes, slashes etc, but the result is always the same.

Is that not the correct way to pass parameters? Do I need the change the dockerfile or something? I'm not very experienced with docker. I can successfully start and use the image without additional parameters

**Comment by gulien** [comment, marked as answer]
You missed the `gotenberg` command 😄 see https://gotenberg.dev/docs/getting-started/installation#modules-configuration

```
docker run --rm -p 3000:3000 gotenberg/gotenberg:7 gotenberg --uno-listener-start-timeout=20s 
```

**Reply by lennartb-** [reply]
Ahh, assumed it was something trivial. Works as expected, thanks 😄
